### ISSUE TITLE
Update Home Page

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#6493]
Want to clean up home page and simplify the Community and Commercial support links/cards.

For the home page:

- Remove the cards on the right-hand side for commercial support
- Remove the 'getting started' card
- Remove the 'what do you want to see...' card
- Rename 'Community Support' as 'Links'
- All links in this 'Links' box should now be configurable
- Add 'Commercial Support' link to the 'Links' card - always shown at the bottom unless commercial support enabled

New settings page - 'Home Links':

- Remove 'Support Links' from 'Branding' settings page'
- The new 'Home Links' should allow user to configure links - Name and URL for each.
- Include the current links as default values
- Add a new setting that stores the links as a JSON blob
- Ensure the old two 'Support Link' settings are migrated to this new world
- These old two settings would no longer be used other than when upgrading
- Add ability to restore the default links, to support the case that the user removes or edits a link and wants to get the default back
- Getting Started should be added as a default link (since the card for it is being removed)
- Defaults:
  - Docs
  - Forums
  - Slack
  - File an Issue
  - Getting Started

**Comment by slickwarren** [comment]
waiting on a new RC to be cut so that we may use a non-docker registry

**Comment by slickwarren** [comment]
when using the `suse` branding, the favicon appears to still be blue. Could we update that? everything else that should be green, is, on the latest tag fo 2.7.0-rc3. 
![Screen Shot 2022-10-07 at 12 55 56 PM](https://user-images.githubusercontent.com/16691014/194643597-b9311bfe-2132-4fe9-9ea3-34defa442acc.png)

**Comment by kwwii** [comment]
The favicon should still be blue. The "SUSE" theme no longer has green as a primary colour.

**Comment by slickwarren** [comment]
that is the only logo that is blue that I could find, is there some rule we're following for when something will be Suse green and when it isn't?

**Comment by nwmac** [comment]
Need to change text for SUSE brand

Need to change colour of favicon

**Comment by slickwarren** [comment]
tested on v2.7.0-rc6:
based testing on original message in this ticket

For the home page:

- [x] Remove the cards on the right-hand side for commercial support
- [x] Remove the 'getting started' card
- [x] Remove the 'what do you want to see...' card
- [x] Rename 'Community Support' as 'Links'
- [x] All links in this 'Links' box should now be configurable -> settings -> `Home Links` - note: existing links are not editable (only shown or not shown)
- [x] Add 'Commercial Support' link to the 'Links' card - always shown at the bottom unless commercial support enabled

New settings page - 'Home Links':

- [x] Remove 'Support Links' from 'Branding' settings page'
- [ ] The new 'Home Links' should allow user to configure links - Name and URL for each. - **FAIL - links and names for default values are not editable** However after testing, you can add a custom link with the exact same name, and this will disable the default value `shown` boolean on `apply`. This is not intuitive, so reopening to add some informational text about how to override an existing value.  
![Screen Shot 2022-10-28 at 10 50 14 AM](https://user-images.githubusercontent.com/16691014/198700585-00f8f4ab-70ec-4347-b438-31a2d769ca37.png)
![Screen Shot 2022-10-28 at 11 25 02 AM](https://user-images.githubusercontent.com/16691014/198707446-887c6832-6d4b-41ec-ac4c-30f518a37e03.png)


- [x] Include the current links as default values
- [ ] Add a new setting that stores the links as a JSON blob -- Where is this setting?
- [ ] Ensure the old two 'Support Link' settings are migrated to this new world 
- [x] These old two settings would no longer be used other than when upgrading - hide all and override issues link is no longer shown (PASS - values are correctly mapped during an upgrade)
- [ ] Add ability to restore the default links, to support the case that the user removes or edits a link and wants to get the default back - FAIL see above screenshot and note on UX for this change. This new flow works, but needs some walkthrough/guidance in the UI to help the user with the new flow. 
- [ ] Getting Started should be added as a default link (since the card for it is being removed) - PASS, but reads `Get Started` not `Getting Started`
- [x] Defaults:
  - [x] Docs
  - [x] Forums
  - [x] Slack
  - [x] File an Issue
  - [ ] Getting Started - PASS, but reads `Get Started` not `Getting Started`
one not listed: support - see https://github.com/rancher/dashboard/issues/6665#issuecomment-1295310248 for more info.

**Comment by Shavindra** [comment]
>  Add a new setting that stores the links as a JSON blob -- Where is this setting?

This is a FE technical implementation comment on how to store the values in BE.

>  Add ability to restore the default links, to support the case that the user removes or edits a link and wants to get the default back - FAIL see above screenshot and note on UX for this change. This new flow works, but needs some walkthrough/guidance in the UI to help the user with the new flow.

This is no longer valid. Flow changed as per the implementation.

>  The new 'Home Links' should allow user to configure links - Name and URL for each. - FAIL - links and names for default values are not editable However after testing, you can add a custom link with the exact same name, and this will disable the default value shown boolean on apply. This is not intuitive, so reopening to add some informational text about how to override an existing value.

This is no longer valid. We didn't want the user to change the text/links of the default links as it would complicate tracking changes user would do over time and makes us harder to implement future changes to links. If the user want a custom link they can add one. If they don't want to show a particular default link they can `show/hide`.

**Comment by slickwarren** [comment]
ok thank you, one more question @Shavindra 
```
Separate note - user is able to use the same method for Commercial Support option to override this variable. Not sure if this is worth disabling override for this specific phrase or if we are OK to leave it as is.
```

**Comment by slickwarren** [comment]
further testing done on v2.7.0-r8 to show the disparity between custom and default links, specifically regarding overriding:


* entering a custom link that does not match an existing display text -> should show up as a new entry on the home page links
* entering a custom link that does match an existing display text  -> currently, there is differing behavior:
  * if the display text matches an existing custom link, **there will be what appears to be a duplicate entry on the home page links, so there appears to be no override available for custom link over custom link** -> this sounds like a bug if overrides are desired. Or, we should check for existing custom links and throw an error that there is already a custom link with the same name if overrides are not desired for custom links. 
  * if the display link matches an existing default link, the link will be overridden with the custom link -> this is the original use case in question that needs clarification in the UI so that users know it is a feature (not a bug)

**Comment by Shavindra** [comment]
> entering a custom link that does match an existing display text -> currently, there is differing behavior:
> entering a custom link that does not match an existing display text -> should show up as a new entry on the home page links

@slickwarren This is implemented as intended. Following requirement changed  
> - Add ability to restore the default links, to support the case that the user removes or edits a link and wants to get the default back

so that user can only show / hide default links and no specific conventions are enforced for custom links which include allowing user to re-enter any link/display text that already exists if thats what they prefer.

**Comment by nwmac** [comment]
@slickwarren We don't want default links to be editable because links like the docs link change between releases (2,6., 2.7 etc). For these we store an id of the link rather than the link itself, so on upgrade, the user will get the latest link.

The user is free to disable a default link and replace it with their own of the same name if they want a custom docs link.

I hope that makes sense.

**Comment by slickwarren** [comment]
Thanks, Neil. After this and some offline discussion, will be closing this ticket in favor of a new ticket.
